.github/workflows/check.yml

```diff
@@ -0,0 +1,21 @@
+name: "Check"
```

Comment:
We have a pretty strong convention that the "main" or "only" CI in a repository lives at `ci.yml` and is named "CI". Any reason not to follow that here?